packages/bson-bench/src/task.ts

```diff
@@ -119,6 +121,7 @@ export class Task {
     const perfSendResults: PerfSendResult = {
       info: {
         test_name: this.testName,
+        tags: this.tags,
```

Comment:
(optional) Is there a reason we're storing `tags` on the class but not `warmup` or `iterations`?

Reply:
Good catch, I'll just pull it off the benchmark object